fishtank-cli/src/commands/start.ts

```diff
@@ -31,7 +43,15 @@ export abstract class Start extends Command {
     const { args, flags } = await this.parse(Start)
     const clusterName = args.name as string
     const cluster = new Cluster({ name: clusterName })
-    await cluster.init({ bootstrap: !flags.noBootstrap })
+
+    if (flags.nodeName !== undefined && flags.nodeImage !== undefined) {
```

Comment:
I think this should be an `||`: one may want to specify an image without specifying a name, and vice-versa.

Reply:
Also maybe this shouldn't be executed if `flags.bootstrap` is false, so maybe this is the correct condition:

```
if (flags.bootstrap && (flags.nodeName !== undefined || flags.nodeImage !== undefined))
```